helix-core/src/main/java/org/apache/helix/controller/GenericHelixController.java

```diff
@@ -1233,4 +1316,24 @@ private void initPipeline(Thread eventThread, BaseControllerDataProvider cache)
     eventThread.setDaemon(true);
     eventThread.start();
   }
+
+  private List<String> getEnabledCustomizedStates(HelixManager manager) {
+    CustomizedStateConfig customizedStateConfig =
+        manager.getConfigAccessor().getCustomizedStateConfig(_clusterName);
+    if (customizedStateConfig != null) {
+      return customizedStateConfig.getAggregationEnabledTypes();
+    }
+    return new ArrayList<>();
```

Comment:
`Collections.singletonList()` may be more lightweight as a returning empty list?